core/build.gradle.kts

```diff
@@ -43,7 +44,7 @@ dependencies {
   testFixturesApi("io.kotest:kotest-assertions-core-jvm:$kotestVersion")
   testFixturesApi("io.kotest:kotest-property-jvm:$kotestVersion")
   testFixturesApi("io.kotest:kotest-assertions-json-jvm:$kotestVersion")
-  testFixturesApi("io.kotest:kotest-assertions-ktor-jvm:4.4.3")
+  testFixturesApi("io.kotest.extensions:kotest-assertions-ktor:2.0.0")
```

Comment:
the kotest assertions from the main repo were still using [now removed] the `ByteReadChannel` class. Modify to the kotest extensions lib